js/app/build_plugins.ts

```diff
@@ -233,6 +233,8 @@ const ignore_list = [
 	"preview",
 	"simpledropdown",
 	"simpletextbox",
+	// "simplevideo",
```

Comment:
Will revert before merge. 

But in order to test these without doing a release, we need to build the app.